enterprise/web.core/src/org/netbeans/modules/web/core/palette/items/IfCustomizer.java

```diff
@@ -188,10 +193,14 @@ private void initComponents() {
 
         getAccessibleContext().setAccessibleDescription(org.openide.util.NbBundle.getMessage(IfCustomizer.class, "ACSD_If_Dialog")); // NOI18N
     }// </editor-fold>//GEN-END:initComponents
+
+    private void jComboBox2ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jComboBox2ActionPerformed
+        // TODO add your handling code here:
+    }//GEN-LAST:event_jComboBox2ActionPerformed
```

Comment:
Why?

Reply:
Don't know exactly what happend. Going to remove that.